WireDomain/Sources/WireDomain/Notifications/Builders/Conversation/ConversationMessageAddEventNotificationBuilder.swift

```diff
@@ -61,7 +61,7 @@ struct ConversationMessageAddEventNotificationBuilder: ConversationMessageAddEve
             )
 
             senderID = mlsMessageEvent.senderID
-            conversationID = mlsMessageEvent.conversationID
+            conversationID = getConversationID(for: message, eventConversationID: mlsMessageEvent.conversationID)
```

Comment:
Do we really need to do this? Don't we know when handling the `UserNotification` if it's a call notification or not? And if so, we can check the `message` to find the call conversation id?